helm/github-exporter-chart/templates/service.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: github-exporter
+  namespace: {{ .Values.namespace }}
+  labels:
+    app: github-exporter
+  annotations:
+    giantswarm.io/exporter: "true"
```

Comment:
Is that right? Should that be a label or annotation?

Reply:
Label, not annotation